README.md

```diff
@@ -14,14 +14,14 @@ Ensure you have:
 2. Move into the directory: `cd SeniorDesign2023BE\backend`
 3. Create a Virtual Environment: `python -m venv venv`
 4. Load the venv: `.\venv\Scripts\activate`
```

Comment:
Doesnt work on macOS

Also got this error in Windows: 
![image](https://github.com/rgreenan55/SeniorDesign2023BE/assets/97375107/e6cb358e-864b-4a92-a7a5-1e73657f1972)
Mention in document if that error occurs run the following line

Reply:
To run it on MAC you would need to use the appropriate activate function for your terminal
![image](https://github.com/rgreenan55/SeniorDesign2023BE/assets/73720654/55b52228-0acc-48bf-8f8c-70872523da97)

for windows it looks like you dont have it enabled to run powershell scripts